nss-day-cohort-60/national-parks-client-v2
# added fetchIt and refactored login/register for token auth

# Description
Deleted unused login.js, modified code on loginModal and resgister.js to accept auth token from server. Renamed np_user to np_token throughout app.

# Steps to test

1. Fetch the JD-login branch and switch to it
2. Start/restart debugger
3. attempt login with various user emails, confirm logged in status (have to refresh page or navigate for navbar welcome message to populate)
4. attempt to register a new user and confirm new user logged in after submit.
*user id, name, token, and staff status should be stored in localStorage on successful logins
